[2022-10-23T23:33:35.446+0000] {processor.py:156} INFO - Started process (PID=27) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:33:35.447+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:33:35.448+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:33:35.447+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:33:35.529+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:33:35.519+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:33:35.541+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:33:35.630+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.186 seconds
[2022-10-23T23:34:05.762+0000] {processor.py:156} INFO - Started process (PID=54) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:34:05.763+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:34:05.763+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:34:05.763+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:34:05.802+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:34:05.792+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:34:05.805+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:34:05.883+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.124 seconds
[2022-10-23T23:34:36.055+0000] {processor.py:156} INFO - Started process (PID=73) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:34:36.055+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:34:36.056+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:34:36.056+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:34:36.091+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:34:36.082+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:34:36.094+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:34:36.174+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.122 seconds
[2022-10-23T23:35:06.549+0000] {processor.py:156} INFO - Started process (PID=101) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:35:06.550+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:35:06.551+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:35:06.550+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:35:06.584+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:35:06.576+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:35:06.586+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:35:06.673+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.126 seconds
[2022-10-23T23:35:36.940+0000] {processor.py:156} INFO - Started process (PID=129) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:35:36.941+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:35:36.942+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:35:36.942+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:35:36.976+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:35:36.968+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:35:36.978+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:35:37.061+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.124 seconds
[2022-10-23T23:37:18.822+0000] {processor.py:156} INFO - Started process (PID=36) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:37:18.823+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:37:18.824+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:37:18.824+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:37:18.928+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:37:18.919+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:37:18.932+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:37:19.049+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.229 seconds
[2022-10-23T23:37:49.175+0000] {processor.py:156} INFO - Started process (PID=55) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:37:49.176+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:37:49.177+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:37:49.177+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:37:49.218+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:37:49.210+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:37:49.222+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:37:49.328+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.156 seconds
[2022-10-23T23:38:19.624+0000] {processor.py:156} INFO - Started process (PID=84) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:38:19.624+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:38:19.625+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:38:19.625+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:38:19.657+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:38:19.650+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:38:19.660+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:38:19.741+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.120 seconds
[2022-10-23T23:38:49.957+0000] {processor.py:156} INFO - Started process (PID=102) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:38:49.958+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:38:49.959+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:38:49.959+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:38:49.999+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:38:49.991+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:38:50.002+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:38:50.086+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.132 seconds
[2022-10-23T23:39:20.309+0000] {processor.py:156} INFO - Started process (PID=129) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:39:20.309+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:39:20.310+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:39:20.310+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:39:20.353+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:39:20.343+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:39:20.357+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:39:20.446+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.140 seconds
[2022-10-23T23:39:50.678+0000] {processor.py:156} INFO - Started process (PID=157) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:39:50.680+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:39:50.681+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:39:50.680+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:39:50.771+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:39:50.762+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:39:50.775+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:39:50.885+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.212 seconds
[2022-10-23T23:40:21.106+0000] {processor.py:156} INFO - Started process (PID=177) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:40:21.107+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:40:21.108+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:40:21.108+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:40:21.148+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:40:21.141+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:40:21.150+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:40:21.239+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.135 seconds
[2022-10-23T23:40:51.555+0000] {processor.py:156} INFO - Started process (PID=205) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:40:51.556+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:40:51.557+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:40:51.556+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:40:51.592+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:40:51.582+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:40:51.595+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:40:51.677+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.124 seconds
[2022-10-23T23:41:21.832+0000] {processor.py:156} INFO - Started process (PID=233) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:41:21.833+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:41:21.834+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:41:21.834+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:41:21.870+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:41:21.860+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:41:21.873+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:41:21.961+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.132 seconds
[2022-10-23T23:41:52.294+0000] {processor.py:156} INFO - Started process (PID=252) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:41:52.295+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:41:52.297+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:41:52.296+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:41:52.339+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:41:52.329+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:41:52.343+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:41:52.430+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.139 seconds
[2022-10-23T23:42:23.013+0000] {processor.py:156} INFO - Started process (PID=280) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:42:23.014+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:42:23.015+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:42:23.015+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:42:23.051+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:42:23.043+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:42:23.054+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:42:23.189+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.179 seconds
[2022-10-23T23:42:53.301+0000] {processor.py:156} INFO - Started process (PID=307) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:42:53.302+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:42:53.303+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:42:53.303+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:42:53.462+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:42:53.456+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:42:53.464+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:42:53.541+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.244 seconds
[2022-10-23T23:43:23.866+0000] {processor.py:156} INFO - Started process (PID=326) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:43:23.867+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:43:23.868+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:43:23.868+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:43:23.910+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:43:23.901+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:43:23.913+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:43:23.999+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.135 seconds
[2022-10-23T23:43:54.266+0000] {processor.py:156} INFO - Started process (PID=356) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:43:54.267+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:43:54.268+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:43:54.268+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:43:54.303+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:43:54.293+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:43:54.305+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:43:54.383+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.119 seconds
[2022-10-23T23:44:24.635+0000] {processor.py:156} INFO - Started process (PID=384) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:44:24.636+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:44:24.637+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:44:24.637+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:44:24.672+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:44:24.664+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:44:24.675+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:44:24.754+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.122 seconds
[2022-10-23T23:44:55.129+0000] {processor.py:156} INFO - Started process (PID=403) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:44:55.130+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:44:55.131+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:44:55.131+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:44:55.170+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:44:55.163+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:44:55.174+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:44:55.277+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.150 seconds
[2022-10-23T23:45:25.424+0000] {processor.py:156} INFO - Started process (PID=430) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:45:25.425+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:45:25.426+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:45:25.425+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:45:25.460+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:45:25.451+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:45:25.462+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:45:25.549+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.128 seconds
[2022-10-23T23:45:55.730+0000] {processor.py:156} INFO - Started process (PID=459) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:45:55.731+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:45:55.732+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:45:55.732+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:45:55.772+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:45:55.763+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:45:55.775+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:45:55.873+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.145 seconds
[2022-10-23T23:46:26.142+0000] {processor.py:156} INFO - Started process (PID=478) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:46:26.143+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:46:26.144+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:46:26.144+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:46:26.212+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:46:26.199+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:46:26.216+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:46:26.362+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.225 seconds
[2022-10-23T23:46:56.450+0000] {processor.py:156} INFO - Started process (PID=506) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:46:56.451+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:46:56.452+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:46:56.452+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:46:56.485+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:46:56.477+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:46:56.488+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:46:56.567+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.120 seconds
[2022-10-23T23:47:27.082+0000] {processor.py:156} INFO - Started process (PID=526) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:47:27.083+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:47:27.084+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:47:27.084+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:47:27.129+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:47:27.119+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:47:27.134+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:47:27.252+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.173 seconds
[2022-10-23T23:47:57.378+0000] {processor.py:156} INFO - Started process (PID=555) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:47:57.379+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:47:57.380+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:47:57.380+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:47:57.414+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:47:57.406+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:47:57.416+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:47:57.503+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.128 seconds
[2022-10-23T23:48:59.842+0000] {processor.py:156} INFO - Started process (PID=27) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:48:59.843+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:48:59.844+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:48:59.844+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:48:59.898+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:48:59.887+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:48:59.903+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:49:00.019+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.179 seconds
[2022-10-23T23:49:30.142+0000] {processor.py:156} INFO - Started process (PID=54) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:49:30.143+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:49:30.143+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:49:30.143+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:49:30.177+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:49:30.169+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:49:30.181+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:49:30.266+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.127 seconds
[2022-10-23T23:50:00.459+0000] {processor.py:156} INFO - Started process (PID=73) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:50:00.460+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:50:00.461+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:50:00.461+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:50:00.501+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:50:00.495+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:50:00.504+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:50:00.605+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.149 seconds
[2022-10-23T23:50:30.777+0000] {processor.py:156} INFO - Started process (PID=101) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:50:30.778+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:50:30.779+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:50:30.779+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:50:30.819+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:50:30.809+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:50:30.823+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:50:30.910+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.136 seconds
[2022-10-23T23:51:01.288+0000] {processor.py:156} INFO - Started process (PID=129) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:51:01.290+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:51:01.291+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:51:01.290+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:51:01.325+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:51:01.318+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:51:01.327+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:51:01.405+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.125 seconds
[2022-10-23T23:51:31.646+0000] {processor.py:156} INFO - Started process (PID=148) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:51:31.648+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:51:31.649+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:51:31.649+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:51:31.694+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:51:31.683+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:51:31.696+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:51:31.778+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.135 seconds
[2022-10-23T23:52:02.050+0000] {processor.py:156} INFO - Started process (PID=177) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:52:02.051+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:52:02.051+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:52:02.051+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:52:02.090+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:52:02.081+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:52:02.094+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:52:02.173+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.125 seconds
[2022-10-23T23:52:32.352+0000] {processor.py:156} INFO - Started process (PID=205) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:52:32.353+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:52:32.354+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:52:32.354+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:52:32.390+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:52:32.382+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:52:32.392+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:52:32.482+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.132 seconds
[2022-10-23T23:53:02.771+0000] {processor.py:156} INFO - Started process (PID=225) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:53:02.771+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:53:02.772+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:53:02.772+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:53:02.804+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:53:02.797+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:53:02.807+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:53:02.893+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.125 seconds
[2022-10-23T23:53:33.117+0000] {processor.py:156} INFO - Started process (PID=253) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:53:33.118+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:53:33.119+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:53:33.119+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:53:33.156+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:53:33.146+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:53:33.159+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:53:33.253+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.139 seconds
[2022-10-23T23:54:03.922+0000] {processor.py:156} INFO - Started process (PID=281) to work on /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:54:03.923+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/custom_dags/finance/Finance.py for tasks to queue
[2022-10-23T23:54:03.924+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:54:03.924+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:54:03.958+0000] {logging_mixin.py:117} INFO - [2022-10-23T23:54:03.950+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/custom_dags/finance/Finance.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/custom_dags/finance/Finance.py", line 11, in <module>
    from dags.common_package import nubankExtractor
ModuleNotFoundError: No module named 'dags'
[2022-10-23T23:54:03.961+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/custom_dags/finance/Finance.py
[2022-10-23T23:54:04.062+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/custom_dags/finance/Finance.py took 0.142 seconds
